src/main/webapp/app/home/home.component.ts

```diff
@@ -165,6 +181,30 @@ export class HomeComponent implements OnInit, AfterViewChecked {
             .finally(() => (this.isSubmittingLogin = false));
     }
 
+    loginSAML2() {
```

Comment:
would it make sense to extract this into a separate component?

Reply:
I am not that familiar with Angular, in this case I would need and appreciate some help.

Reply:
@ivanchimeno could you help out here? You are more familiar with Angular. I guess it makes sense to have multiple components in the home.component. The parent component (home.component) would then include sub components for login, registration or saml2 based on the configuration received from the server. This means we could split up the code nicely and avoid issues

Reply:
I extracted the saml2 login into its own component, but sadly I can't test it because I didn't set up the testing environment for saml2. @LDAP  could you let me know if the changes didn't break anything?